### PR TITLE
Drv db cache 

### PR DESCRIPTION
**Pull request by RobertoAldea**

### CODE REVIEW

File: code/drv_db/src/wattrex_driver_db/drv_db_engine.py [hunk 1]
```diff
@@ -41,38 +38,45 @@ class DrvDbSqlEngineC:
 
     __MAX_RESETS = 2
 
-    def __init__(self, config_file='../config/.cred.yaml'):
+    def __init__(self, db_type : DrvDbTypeE, config_file):
```
Comment (mariuscrsn) [inline code/drv_db/src/wattrex_driver_db/drv_db_engine.py]: add type to config file argument

File: code/drv_db/src/wattrex_driver_db/drv_db_engine.py [hunk 2]
```diff
@@ -41,38 +38,45 @@ class DrvDbSqlEngineC:
...
         '''
-        Create an connector to the MySQL database server
+        Create an connector to the MySQL or SQLite database server
```
Comment (mariuscrsn) [inline code/drv_db/src/wattrex_driver_db/drv_db_engine.py]: It not depends on technology. Distinguishes between master and cache databases.

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_db/tests/example_cache.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+'''
+Enumerations defined to standardize names.
```
Comment (mariuscrsn) [inline code/drv_db/tests/example_cache.py:4]: Test comment for trigger event

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_db/tests/example_cache.py (hunk 3 above)
Comment (mariuscrsn) [reply code/drv_db/tests/example_cache.py:4]: New comment

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_db/tests/example_cache.py (hunk 3 above)
Comment (mariuscrsn) [reply code/drv_db/tests/example_cache.py:4]: New test

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_db/tests/example_cache.py (hunk 3 above)
Comment (mariuscrsn) [reply code/drv_db/tests/example_cache.py:4]: ...

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_db/tests/example_cache.py (hunk 3 above)
Comment (RobertoAldea) [reply code/drv_db/tests/example_cache.py:4]: Solved

File: code/drv_db/src/wattrex_driver_db/drv_db_engine.py (hunk 2 above)
Comment (RobertoAldea) [reply code/drv_db/src/wattrex_driver_db/drv_db_engine.py]: Done

File: code/drv_db/src/wattrex_driver_db/drv_db_engine.py (hunk 1 above)
Comment (RobertoAldea) [reply code/drv_db/src/wattrex_driver_db/drv_db_engine.py]: Done

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**